app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,395 @@
+
+1. Create the `pizza_ordering` service:
+```bash
+    curl --request POST \
+      --url https://<region>.api.konghq.com/v2/runtime-groups/{runtimeGroupId}/core-entities/services \
+      --header 'Authorization: Bearer <personal-access-token>' \
+      --header 'Content-Type: application/json' \
+      --header 'accept: application/json' \
+      --data '{
+          "name": "pizza_ordering",
+          "host": "mockbin.org",
+          "path": "/pizza_ordering"
+      }'
+```
+  You can get the `runtimeGroupId`, by using the [list runtime groups endpoint](https://developer.konghq.com/spec/cd849478-4628-4bc2-abcd-5d8a83d3b5f2/24c1f98b-ea51-4277-9178-ca28a6aa85d9#/Runtime%20Groups/list-runtime-groups) to list all runtime groups and their IDs. <br><br>
+  You should get a `201` response like the following:
+  ```bash
+  {
+      "connect_timeout": 60000,
+      "created_at": 1692885974,
+      "enabled": true,
+      "host": "mockbin.org",
+      "id": "06acc4f4-c6d8-4daf-bef6-79866e88ca86",
+      "name": "pizza_ordering",
+      "path": "/pizza_ordering",
+      "port": 80,
+      "protocol": "http",
+      "read_timeout": 60000,
+      "retries": 5,
+      "updated_at": 1692885974,
+      "write_timeout": 60000
+  }
+  ```
+  Save the `id` value from the output. This is your `GATEWAY-SERVICE-ID` and will be used in another step.
+1. Create the `Pizza Ordering` API product:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/api-products \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --header 'accept: application/json' \
+  --data '{
+      "name": "Pizza Ordering",
+      "description": "API product for pizza ordering"
+  }'
+```
+  You should get a `201` response like the following:
+  ```bash
+  {
+    "labels": {},
+    "id": "0429aa6b-d789-4c73-bbb1-73a228c69477",
+    "name": "Pizza Ordering",
+    "description": "API product for pizza ordering",
+    "portal_ids": [],
+    "created_at": "2023-01-01T00:00:00.000Z",
+    "updated_at": "2023-01-01T00:00:00.000Z"
+  }
+  ```
+  Save the `id` value from the output. This is your `api-product-id` and will be used in another step.
+1. Using the`api-product-id` of the newly created API product, publish the `Pizza Ordering` API product:
+```bash
+curl --request PATCH \
+  --url https://<region>.api.konghq.com/v2/api-products/{apiProductId} \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --header 'accept: application/json' \
+  --data '{
+  "portal_ids":[
+    "DEV_PORTAL_ID"
+  ]
+}'
+```
+  The `DEV_PORTAL_ID` can be found in the {{site.konnect_short_name}} UI in the **Dev Portal** section. <br><br>
```

Comment:
If this link works we should add it

https://cloud.konghq.com/us/portal/published-products